packages/core/frontend/sirius-components-core/src/workbench/WorkbenchMachine.ts

```diff
@@ -77,6 +77,9 @@ const isRepresentationRenamedEventPayload = (
   payload: GQLEditingContextEventPayload
 ): payload is GQLRepresentationRenamedEventPayload => payload.__typename === 'RepresentationRenamedEventPayload';
 
+const equals = (a: Selection, b: Selection) =>
+  a.entries.length === b.entries.length && a.entries.every((v, i) => v.id === b.entries[i].id);
+
```

Comment:
Please make sure to use meaningful variable names instead of variables with one character.